reference/app-availability.html.md.erb

```diff
@@ -0,0 +1,124 @@
+---
+title: App Availability and Resiliency
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+A resilient app is resistant to events like hardware failures or outages. Redundancy, supported by monitoring and scaling, is one of the ways to make your app more resilient, and we provide some built-in features to make this easier.
+
+But first, it's important to understand what your app needs to take care of.
+
+## What your app needs to do
+
+<div class="callout">
+<p><b>A note about Fly Proxy</b>: 
+Fly Proxy does a lot of work in the background, applying handlers, managing traffic and load balancing, and managing connections to services. Fly Proxy also automatically starts and stops Machines based on traffic to your app. The proxy works for [process groups with services defined](/docs/getting-started/app-services/#services-routed-with-fly-proxy).
+</div>
+
+Since Fly Proxy does load balancing by default, you need to consider how your app handles:
+
+* session storage, so that users stay logged in when their session moves to a different server
+* persistent file storage and replication, as discussed in the following paragraphs
+
+For databases, you'll need to follow the recommendations specific to your deployment, including how to set up clusters for replication and failover.
```

Comment:
I don't know if it's just me, but my brain is sliding over "your deployment" as having indeterminate meaning--reading the end of the sentence mitigates that, but I'm lost for a second. Is it recommendations specific to the database? Specific to the desired cluster config?